package.json

```diff
@@ -54,6 +54,7 @@
     "web3": "^1.8.0"
   },
   "dependencies": {
+    "axios": "^1.7.7",
```

Comment:
do we really need axios, we already fetch from  'cross-fetch' you can use that instead of axios so we don't end up having 2 deps with same usage